pallets/capacity/src/lib.rs

```diff
@@ -302,31 +302,32 @@ pub mod pallet {
 		/// Staker attempted to stake to an invalid staking target.
 		InvalidTarget,
 		/// Capacity is not available for the given MSA.
-		InsufficientBalance,
+		InsufficientCapacityBalance,
 		/// Staker is attempting to stake an amount below the minimum amount.
-		InsufficientStakingAmount,
-		/// Staker is attempting to stake a zero amount.
+		StakingAmountBelowMinimum,
+		/// Staker is attempting to stake a zero amount.  DEPRECATED
 		ZeroAmountNotAllowed,
```

Comment:
I don't recall if we can remove this error, but on working in this code again I felt like it was not necessary since we already have `StakingAmountBelowMinimum` which would include zero.